x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/servicenow/helpers.ts

```diff
@@ -22,7 +22,7 @@ export const isRESTApiError = (res: AppInfo | RESTApiError): res is RESTApiError
 export const isFieldInvalid = (
   field: string | undefined | null,
   error: string | IErrorObject | string[]
-): boolean => error !== undefined && error.length > 0 && field != null;
+): boolean => error !== undefined && error.length > 0 && field !== undefined;
```

Comment:
Undefined means the form is untouched, therefore it shows validation errors when the "save" button is clicked.